src/widgets/chart.rs

```diff
@@ -326,6 +498,16 @@ pub struct Chart<'a> {
 }
 
 impl<'a> Chart<'a> {
+    /// Creates a chart with the given [datasets](Dataset)
+    ///
+    /// # Example
+    ///
+    /// Creates an empty chart. See the [struct level doc](Chart) for more details.
```

Comment:
Is this empty? What does the dataset passed here do?

Reply:
It was yes. I thought it was better because filling the points is done and documented on the dataset. Reading this again I feel like I want to know quickly how to create a chart, so I just added the `data` setter.

I also forgot to mention that you can pass multiple datasets, so I added that just now.

What do you think?

Reply:
LGTM